src/features/pages/p2p/hero/banner.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { wrapper, image_style } from './styles.module.scss'
+import Flex from 'features/components/atoms/flex-box'
+
+const DP2HeroBanner = () => {
+    return (
+        <Flex.Box align="end" className={wrapper} pt="34x" md={{ pt: '0x', basis: '6-12' }}>
+            <div className={image_style}>
```

Comment:
```suggestion
            <Flex.Box  className={image_style} align="end" className={wrapper} pt="34x" md={{ pt: '0x', basis: '6-12' }}>
```

Reply:
I didn't understand it.

Reply:
Instead of using classname for div you can use FlexBox.Box and pass them via props